sample_output/rockpi.yml

```diff
@@ -0,0 +1,101 @@
+version: '2'
+
+services:
+
+  gateway-config:
+    image: nebraltd/hm-config:8cc544f
+    depends_on:
+      - dbus-session
+      - diagnostics
+    environment:
+      - FIRMWARE_VERSION=2021.11.22.0-3
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
+    privileged: true
+    network_mode: host
+    cap_add:
+      - NET_ADMIN
+    volumes:
+      - miner-storage:/var/data
+      - dbus:/session/dbus
+    labels:
+      io.balena.features.sysfs: 1
+      io.balena.features.kernel-modules: 1
+      io.balena.features.dbus: 1
+    stop_signal: SIGINT
+
+  packet-forwarder:
+    image: nebraltd/hm-pktfwd:8d8de7d
+    depends_on:
+      - helium-miner
+    restart: always
+    privileged: true
+    volumes:
+      - pktfwdr:/var/pktfwd
+    environment:
+      - FIRMWARE_VERSION=2021.11.22.0-3
+
+  helium-miner:
+    image: nebraltd/hm-miner:arm64-d760384
+    depends_on:
+      - dbus-session
+      - diagnostics
+    expose:
+      - "1680"
+      - "4467"
+    ports:
+      - "44158:44158/tcp"
+    volumes:
+      - miner-storage:/var/data
+      - miner-log:/var/log/miner
+      - pktfwdr:/var/pktfwd
+      - dbus:/session/dbus
+    cap_add:
+      - SYS_RAWIO
+    devices:
+      - /dev/i2c-7:/dev/i2c-7
+    restart: on-failure
+    environment:
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
+      - FIRMWARE_VERSION=2021.11.22.0-3
+
+  diagnostics:
+    image: nebraltd/hm-diag:625ea72
+    environment:
+      - FIRMWARE_VERSION=2021.11.22.0-3
+      - DIAGNOSTICS_VERSION=625ea72
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+    volumes:
+      - pktfwdr:/var/pktfwd
+      - miner-storage:/var/data
+    ports:
+      - "80:5000"
+    cap_add:
+      - SYS_RAWIO
+    devices:
+      - /dev/i2c-7:/dev/i2c-7
+    labels:
+      io.balena.features.sysfs: 1
+      io.balena.features.procfs: 1
+      io.balena.features.dbus: 1
+
+  upnp:
+    image: nebraltd/hm-upnp:b575a2f
+    network_mode: host
+    restart: on-failure
+    volumes:
+      - pktfwdr:/var/pktfwd
```

Comment:
Missing firmware version

Reply:
@SebastianMaj - Basically FIRMWARE_VERSION should be present in all containers as environment variable. This also ensures that the containers restart on firmware upgrade.

Reply:
@kashifpk yes correct

Reply:
@shawaj @kashifpk Oh sorry, I did include FIRMWARE_VERSION under each container, it's just that I didn't generate them locally and push them. Although if you look in the https://github.com/NebraLtd/helium-miner-software/pull/272/files#diff-29d08d4ce056eee6ab4cba496b7a559f37150deb557009f8d3c4fd9fbd3912be you can see that we included it as an environment variable for each